versions.sh

```diff
@@ -83,8 +83,8 @@ for version in "${versions[@]}"; do
 	doc="$(jq <<<"$doc" -c '
 		.variants = [
 			(
+			  "trixie",
 				"bookworm",
```

Comment:
Indentation here is wrong: :eyes:

```suggestion
				"trixie",
				"bookworm",
```